src/main/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchAdminService.java

```diff
@@ -82,15 +85,21 @@ private static String createIndexName() {
     @SneakyThrows
     public boolean slettIndex(String indexName) {
         DeleteIndexRequest deleteIndexRequest = new DeleteIndexRequest(indexName);
-        return restHighLevelClient.indices().delete(deleteIndexRequest, RequestOptions.DEFAULT).isAcknowledged();
+        boolean acknowledged = restHighLevelClient.indices().delete(deleteIndexRequest, RequestOptions.DEFAULT).isAcknowledged();
+        if (!acknowledged) {
+            log.error("Kunne ikke slette index: {}", indexName);
+        }
+        return acknowledged;
     }
 
     @SneakyThrows
     public void opprettAliasForIndeks(String indeks, Boolean isWriteIndex) {
+        BoolQueryBuilder hideAllUsers = QueryBuilders.boolQuery().mustNot(QueryBuilders.existsQuery("fnr"));
```

Comment:
Skal dette alltid skje for `opprettAliasForIndeks`? Bør i tilfelle metodenavn endres?

Reply:
Metodenavnet er endret: opprettSkjultSkriveIndeksPaAlias